bin/create.js

```diff
@@ -324,3 +403,139 @@ function parseHash (argv, field) {
     }, {});
 }
 
+//
+// Load params and secrets from local config files (package.json & .env)
+//
+
+function parseEnv(key) {
+    var secrets = {};
+    var value   = ''
+
+    var noKeyErr = new Error('Non-existant secret specified, must be defined in .env or part of environment');
+
+    // Source from process.env
+    if(key)
+        Object
+            .keys(process.env)
+            .forEach(function (envKey) {
+                if(key === envKey) secrets[key] = process.env[key];
+            });
+
+    // Source from .env
+    var dotenvFile;
+
+    try {
+        dotenvFile = Fs.readFileSync('./.env');
+    } catch(e) {
+        if(key && !secrets[key]) throw noKeyErr
+
+        return secrets;
+    }
+
+    var dotenvObj = dotenv.parse(dotenvFile);
+
+    Object.keys(dotenvObj)
+        .forEach(function (secret) {
+            if(!key) {
+                // Then supply *all* the secrets
+                secrets[secret] = dotenvObj[secret];
+            }
+            else if(secret === key) {
+                value = dotenvObj[secret];
+            }
+        });
+
+    if(key) {
+        if(!value) throw noKeyErr;
```

Comment:
I think that it is plausible that someone might want to set a secret that is an empty string. Perhaps test `typeof value !== 'undefined'`.